Dockerfile

```diff
@@ -1,17 +1,18 @@
-FROM python:3.12-rc-alpine
+FROM python:3.12-alpine3.17
 
 WORKDIR /app
 
 # Define so the script knows not to download a new driver version, as
 # this Docker image already downloads a compatible chromedriver
 ENV AUTO_SOUTHWEST_CHECK_IN_DOCKER=1
 
-RUN apk add --update --no-cache chromium chromium-chromedriver
+RUN apk add --update --no-cache chromium chromium-chromedriver xvfb xauth
 
 RUN adduser -D auto-southwest-check-in -h /app
+RUN chown -R auto-southwest-check-in:auto-southwest-check-in /app
 USER auto-southwest-check-in
 
-COPY requirements.txt requirements.txt
+COPY requirements.txt ./
```

Comment:
What's the reason for this change? AFAIK, this change doesn't have any effect.

Reply:
The ./ notation is often used to indicate a relative path, which aligns with many Docker best practices, especially when you want to make it clear that you're copying to the current directory.